android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/TodoApplication.kt

```diff
@@ -31,5 +34,18 @@ class TodoApplication : Application() {
     override fun onCreate() {
         super.onCreate()
         if (BuildConfig.DEBUG) Timber.plant(DebugTree())
+
+        val deviceNameAndId = android.os.Build.MODEL + Settings.Secure.getString(
```

Comment:
What is the purpose of this exactly? Can we not just use a dummy user id like "dummy-user-id"

Reply:
I wanted to have a unique ID so I can differentiate and use the unique ID for having different rules on them. I will change it to "dummy-user-id".